README.md

```diff
@@ -0,0 +1,23 @@
+Dart Mode
+=========
+Dart Mode is a major mode for editing Dart files in Emacs.
+
+## Installation
```

Comment:
I agree. Link to [this page](https://marmalade-repo.org/#download) and then suggest they run `M-x package-install dart-mode`.